### PR TITLE
[Prototype] Sessions

### PR DESCRIPTION
**Pull request by annaola**
Prototype of sessions, to change after new features.

### CODE REVIEW

File: izp/polls/views.py [hunk 1]
```diff
@@ -16,16 +16,21 @@ def index(request):
 
 def detail(request, question_id):
     question = get_object_or_404(Question, pk=question_id)
+    try:
```
Comment (czapiga) [inline izp/polls/views.py]: I think it can be written without exceptions

File: izp/polls/views.py [hunk 2]
```diff
@@ -78,6 +83,17 @@ def reformat_code(code):
     return newCode
 
 
+def logout(request, question_id):
+    question = get_object_or_404(Question, pk=question_id)
+    try:
+        del request.session[question_id]
+    except KeyError:
```
Comment (czapiga) [inline izp/polls/views.py]: I think it can be written without exceptions

File: izp/polls/views.py [hunk 3]
```diff
@@ -89,14 +105,23 @@ def vote(request, question_id):
...
-    code = reformat_code(code)
-    if code == '' or not question.is_code_correct(code):
-        return render(request,
-                      'polls/detail.html',
-                      {'question': question,
-                       'error': "Niewłaściwy kod uwierzytelniający",
-                       'is_open': is_open})
+    try:
+        code = request.session[question_id]
+    except:
```
Comment (czapiga) [inline izp/polls/views.py]: I think it can be written without exceptions

File: izp/polls/views.py [hunk 4]
```diff
@@ -16,16 +16,21 @@ def index(request):
 
 def detail(request, question_id):
     question = get_object_or_404(Question, pk=question_id)
+    if question_id in request.session:
+        is_session = True
+    else:
+        is_session = False
```
Comment (czapiga) [inline izp/polls/views.py]: Try:
```python
is_session = question_id in request.session
```

Comment (annaola) [reply izp/polls/views.py]: Uh... Indeed.

**Comment by czapiga** [comment]
This PR includes commits that are already in master and it's hard to find what you've changed. Please open new PR that will include only your changes.
Merge my master into yours -> created new branch from your master  and checkout it ->  merge changes from your previous work branch -> create new PR.

**Pull request closed without merging**